templates/panels-development.ini.erb

```diff
@@ -0,0 +1,23 @@
+# THIS FILE IS GENERATED BY PUPPET
+# DO NOT EDIT DIRECTLY, INSTEAD EDIT THE .erb template
+
+<% if @hide_schedule %>
+hide_schedule = <%= @hide_schedule %>
+<% end -%>
+
+<% if @expected_response %>
+expected_response = "<%= @expected_response %>"
+<% end -%>
+
+<% if @event_location %>
+event_location = "<%= @event_location %>"
+<% end -%>
+
+<% if @panel_rooms %>
```

Comment:
Will this work as intended? Keeping in mind that, according to a quick Google search, our empty list is _not_ considered falsey in Ruby.